src/inc/payment-gateways/ppcp-gateway.php

```diff
@@ -4,9 +4,23 @@
 
 use WooCommerce\PayPalCommerce\PPCP;
 use WooCommerce\PayPalCommerce\WcGateway\Gateway\PayPalGateway;
+use Sift_For_WooCommerce\Sift_Events\Events;
 
 add_filter( 'sift_for_woocommerce_ppcp-gateway_payment_gateway_string', fn() => '$paypal' );
 
+add_action(
+	'woocommerce_payments_before_webhook_delivery',
```

Comment:
`ppcp` is using WcPay webhooks structure?

Reply:
Adding to this a bit, because I'm a bit surprised to see this myself. I've been trying to find some clue about where dispute webhooks would be handled with ppcp, and I'm not finding anything. Some things I thought would be related but appear not to be (unless I'm misunderstanding) are:

1. Payment capture reversed - it sounds like chargeback, but it's something separate (`PAYMENT.CAPTURE.REVERSED`).
2. Capture status reason - chargeback / buyer_complaint. This would sound like it's what we want, except that it happens during capture and not via a webhook. I think this is important because I'm thinking this happens when the customer is trying to purchase something and they already have a dispute (though I may be wrong). I would assume that the typical lifecycle of chargebacks means that the customer needs some time to determine that the charge was fraudulent, that it wouldn't happen in the same request as making the purchase, meaning it would need to come in through the webhook.

According to PayPal docs, the webhook event we want to listen to is `CUSTOMER.DISPUTE.CREATED`, and in the admin for this payment gateway, we can see that this event is added when the webhook is created (this gateway allows setting up webhooks from inside wp-admin with the click of a button).

What I'm hoping to find at this point is a generic way to hook into all webhook events for ppcp. I'm not finding anything yet with the search terms I would expect to see (webhook, before, dispute, etc..) It's possible that this plugin may not have implemented these webhooks yet.